src/device-registry/utils/doActivityHelpers.js

```diff
@@ -108,15 +108,15 @@ const doLocationActivity = async (
             });
           } else if (updatedDevice) {
             //then log the operation
-            const log = getModelByTenant(
+            const activityLog = getModelByTenant(
               tenant.toLowerCase(),
               "activity",
               LocationActivitySchema
             ).createLocationActivity(activityBody);
-            log.then((log) => {
+            activityLog.then((activityLog) => {
               return res.status(HTTPStatus.OK).json({
                 message: `${type} successfully carried out`,
-                activityBody,
+                activityLog,
```

Comment:
Please note that the response object for the maintenance has been renamed from `activityBody` to `activityLog`.
@ja-odur

Reply:
Noted